rhizome/minio/bin/configure_minio

```diff
@@ -0,0 +1,21 @@
+#!/bin/env ruby
```

Comment:
Let's use dashes for command names, I think. `configure-minio`, not `configure_minio`. I know there's some programs that break the "rules", but I think it's easier to type maybe? I tend to prefer dashes.